--- ../Metal/meshpreview-vertex.metal	2025-01-26 23:50:24.417867594 -0600
+++ ./meshpreview-vertex.metal.alt	2025-01-26 23:52:20.499617598 -0600
@@ -19,7 +19,7 @@
     float3 in_position [[attribute(0)]];
 };
 
-vertex VS_out VS(VS_in in [[stage_in]], constant ProjectionMatrixBuffer& _16 [[buffer(0)]])
+vertex VS_out VS(VS_in in [[stage_in]], constant ProjectionMatrixBuffer& _16 [[buffer(1)]])
 {
     VS_out out = {};
     out.gl_Position = (_16.projection_matrix * _16.model_matrix) * float4(in.in_position, 1.0);
